notes/returned-mail-received.vbs

```diff
@@ -307,65 +333,73 @@ IF mailing_address_confirmed = "YES" or residential_address_confirmed = "YES" TH
         		EMWriteScreen "NN", 20,79 'cursor is automatically set to 06, 58'
         		TRANSMIT
-        		EMReadScreen MISC_error_msg,  74, 24, 02
-        		IF trim(MISC_error_msg) = "" THEN
-        			case_note_only = FALSE
-        		else
-        			maxis_error_check = MsgBox("*** NOTICE!!!***" & vbNewLine & "Continue to case note only?" & vbNewLine & MISC_error_msg & vbNewLine, vbYESNO + vbQuestion, "Message handling")
-        			IF maxis_error_check = vbNO THEN
-        				case_note_only = FALSE 'this will case note only'
-        			END IF
-        			IF maxis_error_check= vbYES THEN
-        				case_note_only = TRUE 'this will update the panels and case note'
-        			END IF
-        		END IF
         	ELSE
-        		PF9
-        		EMReadScreen open_cash1, 2, 6, 43
-        		EMReadScreen open_cash2, 2, 8, 43
-        		EMReadScreen open_grh, 2, 10, 43
-        		EMReadScreen open_snap, 2, 12, 43
-        		IF cash_active  = TRUE THEN EMWriteScreen "3", 6, 58
-        		IF cash2_active = TRUE THEN EMWriteScreen "3", 8, 58
-        		IF snap_case = TRUE THEN EMWriteScreen "4", 12, 58
-        		IF grh_active = TRUE THEN EMWriteScreen "3", 10, 58
+				PF9
+			END IF
+        	EMReadScreen open_cash1, 2, 6, 43 'I have to read these'
+			EMReadScreen open_cash2, 2, 8, 43
+			IF open_cash1 <> "  " THEN EMWriteScreen "3", 6, 58 'Enter code "3" (Refused/Failed Required Info)'
+			IF open_cash2 <> "  " THEN EMWriteScreen "3", 8, 58 'Enter code "3" (Refused/Failed Required Info)'
+        	If snap_case = True THEN EMWriteScreen "4", 12, 58 'Enter code "4" (Refused/Failed (FS Only))'
+        	IF grh_case = True THEN EMWriteScreen "3", 10, 58 'Enter code "3" (Refused/Failed Required Info)'
+        	TRANSMIT
+        	EMReadScreen pop_upmsg, 7, 11, 08
+        	IF pop_upmsg = "WARNING" THEN
+        		EmWriteScreen "Y", 13, 64 ' this is a pop up box asking if the selection is correct per poli/temp SEE TEMP TE02.13.10'
         		TRANSMIT
-        		EMReadScreen pop_upmsg, 7, 11, 08
-        		IF pop_upmsg = "WARNING" THEN
-        			EmWriteScreen "Y", 13, 64 ' this is a pop up box asking if the selection is correct per poli/temp SEE TEMP TE02.13.10'
-        			TRANSMIT
-        		END IF
         	END IF
 
-        	IF case_note_only = FALSE THEN
-        		IF case_active  = TRUE THEN EMWriteScreen "3", 6, 58
-        		'IF cash2_active = TRUE THEN EMWriteScreen "3", 8, 58 'TODO need to explain this is the reason for reading PROG'
-        		IF SNAP_active  = TRUE THEN EMWriteScreen "4", 12, 58
-        		IF grh_active = TRUE THEN EMWriteScreen "3", 10, 58
-        		TRANSMIT
-        		EMReadScreen pop_upmsg, 7, 11, 08
-        		IF pop_upmsg = "WARNING" THEN
-        			EMWriteScreen "Y", 13, 64 ' this is a pop up box asking if the selection is correct per poli/temp SEE TEMP TE02.13.10'
-        			TRANSMIT
+			EMReadScreen error_message,  74, 24, 02 'for script_run_lowdown'
+	       	IF trim(error_message) = "" THEN
```

Comment:
This is the second time this variable is called. It is used here to some degree, but the repeat variable may cause confusion.